test/middleware.test.js

```diff
@@ -91,7 +91,7 @@ describe('loopback:middleware generator', function() {
         var newSources = Object.keys(
           readMiddlewaresJsonSync('server')['routes:after']);
         var expectedSources = builtinSources.concat(['my-middleware-4']);
-        expect(newSources).to.have.members(expectedSources);
+        expect(expectedSources).to.have.include.members(newSources);
```

Comment:
To be honest, I am confused - it seems that the CI build is passing this test even with the new version of loopback-workspace after https://github.com/strongloop/loopback-workspace/pull/230 was landed.